docs/2.8.0/docs/kubernetes/06-advanced-topics/high-availability.rst

```diff
@@ -1,108 +1,90 @@
-for the Canton Participant.
+You normally achieve high availability through active-passive clustering for the Topology Manager and Mediator components. See the previous section for details on why this is not necessary in a Kubernetes environment. The same principles apply to these components.
 
 Canton Domain: Sequencer
 ========================
 
-We achieve high-availability via active-active clustering for these
-components.
+You achieve high availability through active-active clustering for the Sequencer components. Note the following:
 
--  We deploy 2 replicas by default.
--  We set
-   `stickiness <https://kubernetes.io/docs/concepts/services-networking/service/#session-stickiness>`_
-   based on client IP (via
-   `sessionAffinity <https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity>`_,
-   expires in 1 hour).
+-  Deploy two replicas by default.
+-  Set `stickiness <https://kubernetes.io/docs/concepts/services-networking/service/#session-stickiness>`_ based on the client IP (through `sessionAffinity <https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity>`_; it expires in one hour).
 
-HTTP JSON API Server
+HTTP JSON API server
 ====================
 
-We achieve high-availability via active-active clustering for this
-component.
+You achieve high availability through active-active clustering for the HTTP JSON API server component. Note the following:
 
--  We deploy 2 replicas by default.
--  No `session
-   stickiness <https://kubernetes.io/docs/concepts/services-networking/service/#session-stickiness>`_.
--  Uses ``/livez`` and ``/readyz`` probes (see
-   `documentation <https://docs.daml.com/json-api/index.html#healthcheck-endpoints>`_).
+-  Deploy two replicas by default.
+-  There is no `session stickiness <https://kubernetes.io/docs/concepts/services-networking/service/#session-stickiness>`_.
+-  It uses ``/livez`` and ``/readyz`` probes (see the `HTTP JSON API Service documentation <https://docs.daml.com/json-api/index.html#healthcheck-endpoints>`_).
 
 References
 **********
 
-Here are a few Kubernetes resource types & concepts that are relevant to
-achieving high-availability:
+Here are selected Kubernetes resource types and concepts that are relevant to
+achieve high availability:
 
--  `Service <https://kubernetes.io/docs/concepts/services-networking/service/>`_
+-  `Service method <https://kubernetes.io/docs/concepts/services-networking/service/>`_
 
-   -  Sends workload only to pods that are ready.
+   -  Sends workload only to Pods that are ready.
 
 -  `Deployment <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`_
    >
    `ReplicaSet <https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/>`_
    > `Pods <https://kubernetes.io/docs/concepts/workloads/pods/>`_
 
-   -  Scheduler will try to get to the number of ready Pods you asked
-      for.
+   -  Tells the Scheduler to try to reach the number of ready Pods you request.
```

Comment:
Slight change in meaning, i.e., the below is just a related remark on the above.
```suggestion
   -  The Scheduler tries to reach the number of ready Pods you request at all times.
```